README.md

```diff
@@ -5,11 +5,10 @@
 </p>
 
 <p align="center">
-A minimalistic Flutter game engine.
+The Flutter game engine.
```

Comment:
```suggestion
A Flutter based game engine.
```

Reply:
I think it is too much :P

Reply:
Obviously not having it in caps, but as it is written, still too much?
Then we should come up with something else tag-liney that is shorter.

Reply:
I think we can come up with a better slogan than both of those :P 
Maybe we could pick some core tenants, like:

* modular
* 2d
* flutter-based
* something like easy to use/simple? (which is what minimalistic meant there but I agree that it's not the best words)

and build a short, sweet sentence on that format.

thoughts?

Reply:
That just describes what it is, I think a tag line should be more like the definition of a tag line :smile: :
> a tagline is a short text which serves to clarify a thought for, or is designed with a form of, dramatic effect.

Reply:
I think `serves to clarify a thought` means it should to describe what it is :P 
but I agree that it should be done with flare, spectacle is the part I am bad at hence why I didn't propose anything complete haha
but I think we can use it to include a few core principles we want to highlight

Reply:
Alright, let's drop the `The`, it actually does read a bit weird now when I read it again, could be read as it was the Flutter team making the engine.
Ideally I'd like it short enough to be able to fit on stickers and clothes, underneath the logo.

Reply:
Committed Erick's suggestion until we come up with something else. :)